test/aws-ec2/instance-connect-endpoint.test.ts

```diff
@@ -18,12 +18,20 @@ describe('InstanceConnectEndpoint', () => {
       }),
     });
 
-    Template.fromStack(stack).hasResourceProperties('AWS::EC2::InstanceConnectEndpoint', {
```

Comment:
In general yes, but sometimes I think it is better readable to collapse some KV-pairs